doc/input-output-reference/src/overview/group-electric-load-center-generator.tex

```diff
@@ -1326,8 +1334,8 @@ \subsubsection{Outputs}\label{electricloadcenter-storagebattery-outputs}
 \item
   HVAC,Average, Electric Storage Operating Mode Index {[]}
 \item
-  HVAC,Average,Electric Storage Charge State{[}Ah{]}
-\item
+  HVAC,Average,Electric Storage Charge State{[}Ah{]}  
+\\item
```

Comment:
What happened here?

Reply:
Oops...I forgot to rip that out when I backed things up.  There were a couple of documentation minor edits that I didn't want to lose and then forgot about removing these from the list.  I'll get rid of that/those three.

Reply:
Oops, part 2...I got rid of the old ones but accidentally missed a "\".  Sorry--made a bad assumption without looking carefully at what you referenced.  Anyway, it's gone in the next commit.  Good catch!